cypress/e2e/mini-portfolio/activity-history.test.ts

```diff
@@ -120,7 +120,7 @@ describe('mini-portfolio activity history', () => {
         cy.contains('1 USDC = ').should('exist')
         cy.get('#swap-button').should('not.be', 'disabled').click()
```

Comment:
Actions will not run until their element is enabled: https://docs.cypress.io/guides/core-concepts/interacting-with-elements.

```suggestion
        cy.get('#swap-button').click()
```